src/component/manage_my_tariffs.js

```diff
@@ -18,8 +18,17 @@ export default class ManageMyTariffs extends ViewBase{
     this.filterText = "";
   }
 
+	noTariffsBannerTemplate() {
+		return html`<div class="w3-panel w3-round w3-light-grey">
+			<p>${this.t("manageMyTariffsBanner")}</p>
+			<a href="https://www.mybestchargingcard.com/" target="_blank" class="w3-btn pc-secondary w3-margin-top w3-margin-bottom">Check out now!</a>
+		</div>`
+	}
+
   template(tariffs){
     return html`
+			${!this.myTariffIds.length && this.noTariffsBannerTemplate()}
```

Comment:
This is displayed currently if tariffs are selected:
![image](https://user-images.githubusercontent.com/289532/164470150-0086e8fa-be97-4a0a-9d6e-287049336c41.png)

I think it should be `${this.myTariffIds.length > 0 ? this.noTariffsBannerTemplate() : ""}`

Reply:
Fixed.